### PR TITLE
Geolocation Updated

### PR DESCRIPTION
**Pull request by vchapel**
Added new content to the Geolocation section, about precise & imprecise locations.

**Rendered expanded Geolocation section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-1667#geolocation

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -247,23 +247,25 @@ The family group organizer may stop the data collection from the family safety p
 <!-- ====================================================================== -->
 ## Geolocation
 
-Microsoft Edge supports the [Geolocation API](https://w3.org/TR/geolocation-api), which allows websites to access your location information with your permission.  Websites may ask for your location, for example, when trying to find the closest coffee shop near you.  Microsoft Edge always asks for your permission before granting websites access to your location.  To manage the permission or to always block sites from accessing your location, navigate to `edge://settings/content/location`.
+While you browse the web, websites may request your device’s location from Microsoft Edge. Data about your device’s location can be either precise or imprecise. For example, precise location is requested to provide driving direction to/from your specific location. Imprecise location may be requested to provide search results, news, and weather relevant to your general location.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:250]: change curly quotes to smart quotes, at least in this h2 section
```suggestion
While you browse the web, websites may request your device's location from Microsoft Edge. Data about your device's location can be either precise or imprecise. For example, a precise location is requested to provide driving directions to or from your specific location. An imprecise location may be requested to provide search results, news, and weather relevant to your general location.
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -247,23 +247,25 @@ The family group organizer may stop the data collection from the family safety p
...
+You can allow Microsoft Edge to provide the requesting site with a precise location on Windows 10 and Windows 11. Open **Settings** > **Privacy** > **Location** and turn on the **Allow access to location on this device** and **Allow apps to access your location** settings.
 
- If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, Microsoft Edge provides an approximate location to the requesting site. The information is only shared with a requesting site if you previously allowed sharing of your location. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
+ If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites can use information such as your IP address to determine your device’s imprecise location. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:264]:
```suggestion
 If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites can use information such as your IP address to determine your device's imprecise location. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -247,23 +247,25 @@ The family group organizer may stop the data collection from the family safety p
...
+While you browse the web, websites may request your device’s location from Microsoft Edge. Data about your device’s location can be either precise or imprecise. For example, precise location is requested to provide driving direction to/from your specific location. Imprecise location may be requested to provide search results, news, and weather relevant to your general location.
 
-On the right side of the address bar, Microsoft Edge indicates when your location is or is not being shared.
+Microsoft Edge supports the [Geolocation API](https://w3.org/TR/geolocation-api), which allows websites to access your precise location with your permission. Microsoft Edge always asks for your permission before granting websites access to your precise location. To manage the site-specific permissions or to always block sites from accessing your precise location, navigate to `edge://settings/content/location`.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:252]:
```suggestion
Microsoft Edge supports the [Geolocation API](https://w3.org/TR/geolocation-api), which allows websites to access your precise location with your permission. Microsoft Edge always asks for your permission before granting websites access to your precise location. To manage the site-specific permissions or to always block sites from accessing your precise location, go to `edge://settings/content/location`.
```
Add UI steps as an alternative to the special-page URL?

File: microsoft-edge/privacy-whitepaper/index.md [hunk 4]
```diff
@@ -247,23 +247,25 @@ The family group organizer may stop the data collection from the family safety p
...
 
- If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, Microsoft Edge provides an approximate location to the requesting site. The information is only shared with a requesting site if you previously allowed sharing of your location. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
+ If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites can use information such as your IP address to determine your device’s imprecise location. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
 
-Microsoft Edge generates a new random ID when making requests to the location service.  Microsoft Edge location service does not store your geolocation coordinates.
+Microsoft Edge does not store your geolocation coordinates. When making requests to the Microsoft location service, Microsoft Edge generates a new random ID for each request.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:266]:
```suggestion
Microsoft Edge doesn't store your geolocation coordinates. When making requests to the Microsoft location service, Microsoft Edge generates a new random ID for each request.
```

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:252]: Add UI steps as an alternative to the special-page URL?

**Comment by dalenaMSFT** [comment]
+ @Laura ***@***.***>

Michael, is there a question here (UI steps?), or is this just a request for review?
________________________________

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:252]: Sent an email about this including Laura, she can't see this page.

Comment (lchambersMSFT) [reply microsoft-edge/privacy-whitepaper/index.md:252]: Looks like I have access to this page now. Did a quick scan of the article and the special-page URL is used commonly, so my suggestion is to stick with that.

Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:252]: Done

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:250]: Changed in a single Commit

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:264]: Changed in a single Commit

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:252]: Changed in a single Commit

File: microsoft-edge/privacy-whitepaper/index.md (hunk 4 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:266]: Changed in a single Commit

**Review by lchambersMSFT — APPROVED** [review]
Looks good to me, thanks for making the update.

**Review by dalenaMSFT — APPROVED** [review]

**Review by dalenaMSFT — APPROVED** [review]

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**